addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -16,31 +16,42 @@ Note that the commands are executed in the context and with the privileges of th
 
 The `command` Thing requires the command to execute on the shell. Optionally one can specify:
 
-- a transformation to apply on the execution result, 
-- an interval, in seconds, the command will be repeatedly executed, 
-- a time-out, in seconds, the execution of the command will time out, and lastly, 
-- a boolean parameter to make the command execute immediately every time the state of the input channel has changed. 
+- [transform] a transformation to apply on the execution result, 
+- [interval] an interval, in seconds, the command will be repeatedly executed, 
+- [timeput] a time-out, in seconds, the execution of the command will time out, and lastly, 
+- [runOnInput] a boolean parameter to make the command execute immediately every time the state of an input channel is set by a Command 
```

Comment:
> It also could lessen potential confusion about a "run" command using the previous command sent to the input channel rather than its current state.

Beware that "run" does not use the previous command sent to the input, but last command that was sent. e.g. you can send ON to run at all times even if you do not use an input channel

If the objective would be to work with the most recent state of the channel (and thus the Item), then we are better of implementing the handleUpdate() as well (e.g set the lastInput when the Item changes state)